//Aula 68 -  Exceptions pt 02
package br.com.abc.javacore.parte16_excecoes.runtimeexception.test;

public class RuntimeException
{
	public static void main(String[] args)
	{
		// int a = 10 / 0;
		// System.out.println(a);
		// Erro em tempo de execução, já que esse valor nulo poderia ter vindo de um
		// banco de dados ou digitado pelo usuário e que deve ser tratado.
		//
		// Object o = null;
		// // Erro de uma referência para nenhum objeto
		// System.out.println(o.toString());
		//
		// Fora da faixa do vetor.
		// int [] a = new int[2];
		// System.out.println(a[2]);
	}
}
